api/src/main/java/org/eclipse/microprofile/reactive/messaging/Acknowledgment.java

```diff
@@ -87,10 +57,11 @@
     PRE_PROCESSING,
 
     /**
-     * Acknowledgment performed automatically after the user processing of the message.
+     * Acknowledgment performed automatically after the processing of the message.
+     * When {@code POST_PROCESSING} is used, the incoming message is acknowledged when the produced message is
+     * acknowledged.
      *
-     * Notice that this mode is not supported for all signatures. check the list above.
-     * When supported it's the default policy.
+     * Notice that this mode is not supported for all signatures. When supported it's the default policy.
```

Comment:
When support it's the default policy => When support, it's the default policy.

Reply:
You can't really say "When support, it's the default policy". (support is the noun, supported is the adjective.)

Reply:
I used "When supported," <- the coma was missing.